config/util/loggers_test.go

```diff
@@ -0,0 +1,31 @@
+package util
+
+import (
+	"bytes"
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var expected string = `This is test line 2
+This is test line 3
+`
+
+func TestLogRandomSample(t *testing.T) {
```

Comment:
Now that we have extracted LogRandomSample, you can test the `logRandomSampleImpl` method with a mocked constant value and only need to test the two conditions of 'log' or 'ignore'.

Reply:
True, but is it worth redoing the test to break it up into those two separate cases?